clang/include/clang/CIR/Dialect/IR/CIROps.td

```diff
@@ -3497,6 +3497,17 @@ def LLVMIntrinsicCallOp : CIR_Op<"llvm.intrinsic"> {
 
 }
 
+//===----------------------------------------------------------------------===//
+// DeleteArrayOp
+//===----------------------------------------------------------------------===//
+
+def DeleteArrayOp : CIR_Op<"delete.array">,
+              Arguments<(ins CIR_PointerType:$address)> {
+  let summary = "Delete address representing an array";
```

Comment:
Can you add an example as part of the `description`?

Reply:
I think I added the example in the description.